web/src/app/components/resources/list/list.component.ts

```diff
@@ -25,33 +30,45 @@ export class ListComponent implements AfterViewInit {
   ];
   @ViewChild(MatPaginator) paginator: MatPaginator;
   @ViewChild(MatSort) sort: MatSort;
-  filterEmitter = new EventEmitter();
 
-  resources: EsHits<Resource>[] = [];
+  resources: Observable<EsHits<Resource>[]>;
```

Comment:
`$` suffix is used to indicate observable.

Reply:
done